src/cmd/cmd_wasm2wat.ml

```diff
@@ -0,0 +1,15 @@
+(* SPDX-License-Identifier: AGPL-3.0-or-later *)
+(* Copyright © 2021-2024 OCamlPro *)
+(* Written by the Owi programmers *)
+
+open Syntax
+
+let cmd_one file =
+  let ext = Fpath.get_ext file in
+  match ext with
+  | ".wasm" ->
+    let* m = Binary_deserializer.from_file file in
+    Ok (Format.pp_std "%a@\n" Simplified.Pp.modul m)
+  | ext -> Error (`Msg (Format.sprintf "invalid extension: `%s`" ext))
```

Comment:
Debatable we want to enforce extension but not blocking for merge.
We could instead check the magic bytes.

Reply:
Yes, I'm not sure yet what's best. I guess it's better to see how we handle this in other places (parse.ml, compile.ml, owi sym, owi run, owi opt etc.) later and do the same everywhere